sql/moz-fx-data-shared-prod/telemetry_derived/events_last_seen_v1/query.sql

```diff
@@ -0,0 +1,59 @@
+WITH _current AS (
+  SELECT
+    submission_date,
+    sample_id,
+    client_id,
+    -- In this raw table, we capture the history of activity over the past
+    -- 28 days for each usage criterion as a single 64-bit integer. The
+    -- rightmost bit represents whether the user was active in the current day.
+    CAST(TRUE AS INT64) AS days_logged_event_bits,
+    CAST(
+      LOGICAL_OR(event_method = 'show' AND event_object = 'protection_report') AS INT64
+    ) AS days_viewed_protection_report_bits,
+    CAST(
+      LOGICAL_OR(event_category = 'pictureinpicture' AND event_method = 'create') AS INT64
+    ) AS days_used_picture_in_picture_bits,
+    CAST(
+      LOGICAL_OR(event_string_value = 'SEC_ERROR_UNKNOWN_ISSUER') AS INT64
+    ) AS days_had_cert_error_bits,
+  FROM
+    telemetry.events
+  WHERE
+    submission_date = @submission_date
+),
```

Comment:
@fbertsch Am I correct that this whole `events_last_seen_v1` proposed here is essentially redundant with the proposed `events_daily` and the 28-day rollups? Likely, we can serve the need for joining event-based usage criteria into clients_last_seen_v1 by a join on the 28-day rollup table rather than maintaining something separate.

Is there a timeline for that being available for desktop?